### DISCUSSION TITLE
Resolving Inconsistencies: Blue Shadow Persisting in Progress Bar Despite Color

### DISCUSSION THREAD

**Discussion by masihgh** [livewire/livewire#7544] [7. Bugs (archived)] [answered]
### Livewire version

v3.3.0

### Laravel version

v10.10

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

In Livewire configuration, there appears to be an issue where the progress bar retains a blue shadow despite attempts to change its color. Could you look into resolving this inconsistency?

### Repository or Wirebox URL

#

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
@masihgh PRs to fix the issue are welcome 🙂
